config/config.go

```diff
@@ -476,6 +477,11 @@ func (t *TLSConfig) Validate() error {
 	return nil
 }
 
+type Metrics struct {
+	NoAuth  bool `toml:"no_auth" json:"no-auth"`
```

Comment:
What do you think about the following defaults for metrics:

  * By default they are disabled
  * Authentication is enabled by default

In which case the settings can look something like this:

```suggestion
	DisableAuth  bool `toml:"disable_auth" json:"disable-auth"`
	Enabled        bool `toml:"enabled" json:"enabled"`
```

That way, if this section is completely omitted,  defaults should be good.

Considering this is just a couple of boolean values, you're right, makes no sense to validate.

Reply:
A config would look something like this:

```toml
[metrics]
enabled = true
disable_auth = false
```